src/entities/event/api/event.ts

```
@@ -0,0 +1,71 @@
+};
+
+// 이벤트 생성 (POST)
+export const createEvent = async (eventData: EventCreate): Promise<EventResponse<string>> => {
+  const response = await axiosClient.post<EventResponse<string>>(`/events`, eventData);
+  return response.data;
+};
+
+// 이벤트 수정 (PUT)
+updateEvent: async (eventId: number, eventData: EventUpdate): Promise<EventResponse<string>> => {
+  const response = await axiosClient.put<EventResponse<string>>(`/events/${eventId}`, eventData);
+  return response.data;
```

Biome (1.9.4)

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)